### ISSUE TITLE
Error with lock file creation

### ISSUE THREAD

**Issue by mbrodala** [pnpm/pnpm#849]
### pnpm version:
1.6.0

### Code to reproduce the issue:
Run `pnpm i` (in `/var/www` here)

Content of pnpm-debug.log file:

```
---
  0 error pnpm:
    message:
      errno: -13
      code: "EACCES"
      syscall: "mkdir"
      path: "/var/www.lock.lock"
    err:
      name: "Error"
      message: "EACCES: permission denied, mkdir '/var/www.lock.lock'"
      code: "EACCES"
      stack: "Error: EACCES: permission denied, mkdir '/var/www.lock.lock'\n    at Error (native)"
```

### Expected behavior:
Packages are installed as usual.

### Actual behavior:
An error occurs:

```
ERROR EACCES: permission denied, mkdir '/var/www.lock.lock'
```

I have to run with `--no-lock` to bypass this.

### Additional information:

 - `node -v` prints: v5.12.0
 - Windows, OS X, or Linux?: Linux

**Comment by mbrodala** [comment]
Seems like [version 1.5.3](https://github.com/pnpm/pnpm/releases/tag/v1.5.3) broke this, reverting to version 1.5.2 resolves the issue for now:

```
$ sudo npm i -g pnpm@1.5.3
/usr/bin/pnpm -> /usr/lib/node_modules/pnpm/lib/bin/pnpm.js
/usr/lib
`-- pnpm@1.5.3

$ pnpm i
ERROR EACCES: permission denied, mkdir '/var/www.lock.lock'

$ sudo npm i -g pnpm@1.5.2
/usr/bin/pnpm -> /usr/lib/node_modules/pnpm/lib/bin/pnpm.js
/usr/lib
`-- pnpm@1.5.2

$ pnpm i
Resolving: total 0, reused 0, downloaded 0, done
Creating dependency tree
```

**Comment by zkochan** [comment]
That is useful information, thanks

**Comment by zkochan** [comment]
Before version 1.5.3 the lock would have been created in `/var/www/.lock.lock`. In version 1.5.3 it was changed to create the lock in the `/var/` directory. Seems like on your system pnpm does not have permission to create a folder in the directory.

What should be the fix? We can either revert and think about a different solution to #843. Or we can use a central location for all the lockfiles. I think that is how npm does it. I like the second solution more.

**Comment by zkochan** [comment]
Publish fixed in [v1.6.1](https://github.com/pnpm/pnpm/releases/tag/v1.6.1)

**Comment by mbrodala** [comment]
@zkochan Thanks, the solution looks good and works like a charm:

```
$ sudo npm i -g pnpm
/usr/bin/pnpm -> /usr/lib/node_modules/pnpm/lib/bin/pnpm.js
/usr/lib
`-- pnpm@1.7.0

$ pnpm i
Resolving: total 0, reused 0, downloaded 0, done
Creating dependency tree
```
